### ISSUE TITLE
False positive of version pinning for apt-get

### ISSUE THREAD

**Issue by bqcuong** [hadolint/hadolint#928]
- [x] This is a bug report
- [x] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
There should not be any warnings on the version pinning.

### Actual behavior
There is a false positive warning of the version pinning (DL3008) when I try to run Hadolint on the Dockerfile below..
Output from the run:
```
Dockefile:2 DL3008 warning: Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`
```
### Steps to reproduce the behavior

Dear Hadolint team,

I got the false positive result for version pinning of apt-get command.
Please take a look at the Dockerfile below, here the package `python3` and its version information are passed to `apt-get` by the variable `$buildDeps`.
Hadolint may think that `$buildDeps` is a package without its version information and raises the alert.
I am not sure if this is a bug or a feature that Hadolint hasn't implemented yet, so I mark both the checkboxes.

Best,
Cuong

**Output of `hadolint --version`**

```bash
Haskell Dockerfile Linter 2.12.0
```

**Dockerfile** (if relevant)

```Dockerfile
FROM ubuntu:18.04
RUN apt-get update \
 && buildDeps="python3=3.6.7-1~18.04" \
 && apt-get install -y --no-install-recommends $buildDeps \
 && apt-get clean \
 && rm -rf /var/lib/apt/lists/*
```

**Comment by bqcuong** [comment]
Close due to duplicated with #363.
